### PR TITLE
USBL coin ui merge please

### PR DESCRIPTION
**Pull request by softbalanced**
added coins.json, image, qml

### CODE REVIEW

File: assets/config/0.5.1-coins.json [hunk 1]
```diff
@@ -5266,6 +5266,25 @@
...
+  "USBL": {
+    "active": true,
+    "coin": "USBL",
+    "name": "Balanced",
+    "coingecko_id": "balanced",
+    "coinpaprika_id": "balanced-balanced",
+    "currently_enabled": true,
+    "electrum": [
+      {
+        "url": "",
```
Comment (smk762) [inline assets/config/0.5.1-coins.json:5278]: A coin can not be added without an electrum, or else transactions / balances will not be functional.

**Review by smk762 — COMMENTED** [review]
To be tradable within the AtomicDEX network, details need to be added also to this repo - https://github.com/KomodoPlatform/coins including txids of an example swap.

**Comment by tonymorony** [comment]
thanks for the PR @softbalanced !
please check coin adding guidelines and compatibility check here first: https://github.com/KomodoPlatform/coins#0-the-coin-must-have-participated-in-a-successful-atomic-swap-using-atomicdex-api
then add your coin into coins repo if it's compatible and reopen clean PR into desktop GUI after your coin adding into coins repo

File: assets/config/0.5.1-coins.json (hunk 1 above)
Comment (softbalanced) [reply assets/config/0.5.1-coins.json:5278]: electrum urls updated with pull #323

**Pull request closed without merging**